extensions/GitExtension/FileExplorerGitIntegration/FileExplorerGitIntegration.csproj

```diff
@@ -17,7 +17,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="LibGit2Sharp" Version="0.29.0" />
+      <PackageReference Include="LibGit2Sharp" Version="0.30.0" />

```

Comment:
Does the GitHubextension and Azure Extension have PR's to upgrade to the same version?

Reply:
It would be good for them to update when appropriate for them to do so, but it shouldn't be a requirement, since they're separate packages/extensions, yes?